runtime/plugin/fcnotify.vim

```diff
@@ -0,0 +1,10 @@
+" autocommands for starting filesystem based file watcher
+
+augroup fcnotify
```

Comment:
Is there any way this augroup can be defined _only_ when someone is using the setting?

That would make it easier for people to completely turn off the feature (if they wanted).

Reply:
I think this should be possible with autocmd `OptionSet`. I'll give this a shot today.

Reply:
Did this happen yet? Just checking back in on this PR

Reply:
@tjdevries, it didn't :(
I went on a trip and just returned. Sorry for not informing you about this earlier. However, now that I have had a good break, I'll get back on it immediately.